packages/config/config/devices/0x0173/lgrm.json

```diff
@@ -50,7 +50,7 @@
 		"3": {
 			"label": "Report Rate Interval (Association Group 1)",
 			"valueSize": 1,
-			"unit": "6 minutes",
+			"unit": "6 min per count",
```

Comment:
That's not actually better. Is it really the configured value times 6? In that case, the original was okay.
Or: `x6 minutes` 🤷🏻‍♂️

Reply:
The value is 1/10th of an hour, so 1 means 6 minuntes... 10/1*60=6 minutes.

Reply:
Okay then lets stick with `6 minutes` i guess.